alerts/machine-config-operator/MachineConfigControllerPoolAlert.md

```diff
@@ -0,0 +1,73 @@
+# MachineConfigControllerPoolAlert
+
+## Meaning
+
+Alerts the user to when the Machine Config
+Controller (MCC) detects when nodes are
+apart of the master pool and a custom pool.
+Causing overlapping
+Machine Config Pools (MCP) to apply.
```

Comment:
nit: they don't both apply, so maybe explicitly say `Causing the custom pool configuration to not be applied for those nodes` ?